docs/source/contributing/documentation.rst

```diff
@@ -140,9 +214,10 @@ See an example on list item 4.
 
        # python code here
 
-**NOTE: Also, if this is a video- or animation-related change, please
-try to add an example GIF or video if possible for demonstration
-purposes.**
+.. NOTE::
```

Comment:
```suggestion
.. NOTE::

```

Reply:
I think it should be .. note::
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
I *think* it doesn't matter, but I'm fine either way.